tutorial/quickstart.md

```diff
@@ -0,0 +1,462 @@
+# Pyccel's Quickstart Guide
+
+Pyccel is a **static compiler** for Python 3, using Fortran or C as a backend language, with a focus on high-performance computing (HPC) applications.
+
+Pyccel's main goal is to resolve the main bottleneck in scientific computing, that is the transition from **prototype** to **production**.
```

Comment:
This line sounds strange due to the repetition of the word `main` and the grammar around `that`. I suggest:
```
Pyccel's main goal is to resolve the principal bottleneck in scientific computing: the transition from **prototype** to **production**
```

Reply:
Fixed in commit eb26349